src/index.js

```diff
@@ -0,0 +1,54 @@
+import './styles.css';
+
+// ### 1. Data
+const taskList = [
+  {
+    description: 'Shopping',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'Study ES6 - Webpack',
+    completed: true,
+    index: 1,
+  },
+  {
+    description: 'Complete Awesome Books ES6',
+    completed: false,
+    index: 2,
+  },
+
+  {
+    description: 'Workout',
+    completed: true,
+    index: 3,
+  },
+];
+
+// ### 2. DOM Manipulations
+const mainContainer = document.querySelector('.todo-list-container');
+
+mainContainer.innerHTML = `<div class="row">
+<h1>Today's To Do</h1>
+<i class="fa-solid fa-rotate fa-lg font-awesome-icon"></i>
+</div>
+<div>
+<input placeholder="Add to your list...">
+</div>`;
+
+taskList.forEach((e) => {
+  let isChecked;
+  let strikeThrough;
+  if (e.completed === true) {
+    isChecked = 'checked';
+    strikeThrough = 'strike-through';
+  }
+
+  mainContainer.innerHTML += `<div class="row">
+  <input class="checkbox" type="checkbox" ${isChecked}>
+  <p class="${strikeThrough}">${e.description}</p>
+  <i class="fa-solid fa-ellipsis-vertical fa-lg font-awesome-icon"></i>
+  </div>`;
+});
```

Comment:
The functionality that iterates over the tasks array and populates the HTML list item element for each task must be a `function`. Please wrap your code inside a function:

You must have something similar to this:
![image](https://user-images.githubusercontent.com/38088818/167001127-d2a8af9a-7554-40e6-bc4a-dd5c45ed7a89.png)

- This is just an example of how it can be implemented but you can still use your own code. The main goal is to make it a `function`.